demo/style.css

```diff
@@ -155,6 +155,341 @@ h6 {
   margin: 0 24px;
 }
 
+.ProseMirror {
+  position: relative;
+}
+
+.ProseMirror {
+  word-wrap: break-word;
+  white-space: pre-wrap;
+  white-space: break-spaces;
+  -webkit-font-variant-ligatures: none;
+  font-variant-ligatures: none;
+  font-feature-settings: "liga" 0; /* the above doesn't seem to work in Edge */
```

Comment:
You're right to highlight issues of cross-browser compatibility, as this only works in 2 specific versions of Edge: https://caniuse.com/?search=font-feature-settings